authentication/authentication.py

```diff
@@ -0,0 +1,19 @@
+from django.utils.timezone import now
+from rest_framework.authentication import TokenAuthentication
+from rest_framework.exceptions import AuthenticationFailed
+from rest_framework.authtoken.models import Token
+
+from forum import settings
```

Comment:
*must* be:

```
from django.conf import settings
```

Please share with the rest of the team.

Reply:
Fixed